scenes/cslib_string.txt

```diff
@@ -203,10 +203,10 @@
 *comment Inputs of a length not equal to 1 will cause an error to be thrown.
 *comment ------------------------------
 *comment 	params:
-*comment		p_str(string): a string to search for p_char
-*comment		p_char(string): a string of length 1
+*comment		p_str(string): a string in which to search for p_char
```

Comment:
consider: in which -> where

Reply:
Hmm... That also doesn't sound quite right to me. What about just "_a string to search for p_char_"?